python/ray/data/_internal/execution/interfaces/op_runtime_metrics.py

```diff
@@ -475,6 +529,7 @@ def on_task_output_generated(self, task_index: int, output: RefBundle):
 
         self.num_task_outputs_generated += num_outputs
         self.bytes_task_outputs_generated += output_bytes
+        self.average_bytes_task_outputs_generated = int(self.bytes_task_outputs_generated / self.num_task_outputs_generated)
```

Comment:
This is only used for stats and not other estimation for scheduling right?

Reply:
Yes this is only used for stats.